src/main/java/com/mariamkatamashvili/trainerwork/entity/TrainersWork.java

```diff
@@ -0,0 +1,50 @@
+package com.mariamkatamashvili.trainerwork.entity;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Objects;
+import java.util.Set;
+
+@Data
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class TrainersWork {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String username;
+    private String firstName;
+    private String lastName;
+    private Boolean status;
+
+    @OneToMany(mappedBy = "trainersWork", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    @JsonManagedReference
+    private Set<Years> years;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        TrainersWork that = (TrainersWork) o;
+        return Objects.equals(id, that.id) &&
+                Objects.equals(username, that.username);
+    }
```

Comment:
Is it possible to serve that with lombok annotation? If that needs extra management then a separate annotation Equals with additional attributes could be used